mirage/seed_image/catalog_seed_image.py

```diff
@@ -2457,7 +2465,7 @@ def make_point_source_image(self, pointSources, segment_number=None, ptsrc_segma
 
             scaled_psf, min_x, min_y, wings_added = self.create_psf_stamp(
                 entry['pixelx'], entry['pixely'], psf_x_dim, psf_y_dim,
-                segment_number=segment_number
+                segment_number=segment_number, ignore_detector=True
```

Comment:
This is the key fix to get the point sources to appear in the extended area around the detector.